database/db.go

```diff
@@ -57,18 +62,45 @@ func (d dbLogger) AfterQuery(q *pg.QueryEvent) {
 	d.logger.Printf("executed query\n%+v ", query)
 }
 
-// New returns a new DB object which wraps
-// a connection to the database specified in config
+// New returns a new DB object which wraps a connection to the database specified in config
 func New(config DBConfig) DB {
 	db := pg.Connect(&pg.Options{
 		Addr:     config.Address,
 		User:     config.User,
 		Database: config.Database,
 		Password: config.Password,
 	})
-	if os.Getenv("QUERY_DEBUG") == "ON" {
-		db.AddQueryHook(dbLogger{logger: log.New(os.Stdout, fmt.Sprintf("%s: %s: ", config.Address, config.Database), log.LstdFlags)})
+	if config.EnableLogging {
```

Comment:
Trying to think about how the Service Logger can be integrated into this, so we don't have to specifically enable at the DB level... But everything I can think of is slightly convolute